docs/connector-development/config-based/understanding-the-yaml-file/reference.md

```diff
@@ -1,3 +1,50 @@
+import ManifestYamlDefinitions from '@site/src/components/ManifestYamlDefinitions';
+
+import schema from "../../../../airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml";
+
+
 # YAML Reference
 
-To see every component and each possible parameter, see the full YAML schema reference [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml).
+This page lists all components, interpolation variables and interpolation macros that can be used when defining a low code YAML file.
+
+For the technical JSON schema definition that low code manifests are validated against, see [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml).
```

Comment:
Should we link to the $parameters section since it's a field in all the components?

Maybe something like

> All components have an optional `$parameters` field used to inherit propertyies. See the section in the advanced topics for more details (../advanced-topics.md#parameters).

Reply:
Good idea, added that:
<img width="866" alt="Screenshot 2023-04-25 at 10 54 08" src="https://user-images.githubusercontent.com/1508364/234226270-7adafa4b-feb9-4e09-a40a-67f40d72986d.png">